__tests__/app.test.js

```diff
@@ -0,0 +1,47 @@
+const request = require('supertest')
+const app = require('../app')
+const connection = require('../db/connection')
+const testData = require('../db/data/test-data')
+const seed = require('../db/seeds/seed')
+
+afterAll(() => {
+    return connection.end()
+  })
+  
+beforeEach(() => {
+    return seed(testData)
+})
+
+describe('endpoint GET /api/topics',()=>{
+    test('receives 200 response', ()=>{
```

Comment:
This test's now a bit redundant because you test the status code in the test below, so you could remove it